core/src/main/kotlin/com/justai/jaicf/channel/http/HttpBotResponse.kt

```diff
@@ -12,19 +12,61 @@ import java.nio.charset.StandardCharsets
  */
 data class HttpBotResponse(
     val output: ByteArrayOutputStream,
-    val contentType: String
+    val contentType: ContentType,
+    val statusCode: HttpStatusCode = HttpStatusCode.OK
```

Comment:
I'd suggest making statusCode a simple Integer. There is no point in limiting codes to our enum.